example/package.json

```diff
@@ -63,8 +64,8 @@
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
     "react-app-rewired": "^2.2.1",
-    "react-scripts": "^5.0.1",
-    "serve": "^13.0.2",
+    "react-scripts": "^2.1.3",
+    "serve": "^14.2.0",
```

Comment:
I'm not sure that we need to update libraries. Could you please explain why it was done?